harness-util/package-util.sh

```diff
@@ -2,28 +2,27 @@
 
 set -eu
 
-declare -r DIST_PACKAGE_NAME="tic-tac-toe"
+declare -r APP_NAME="tic-tac-toe"
 
-# ${1} - the name of the target OS
-# ${2} - the architecture of the target OS
+# ${1} - the target OS
+# ${2} - the target ISA
 function pkg_src_dir() {
     declare -r BUILD_DIR_NAME="build"
-    echo "./${BUILD_DIR_NAME}/${DIST_PACKAGE_NAME}_${1}_${2}"
+    echo "./${BUILD_DIR_NAME}/${APP_NAME}_${1}_${2}"
 }
 
-# ${1} - the name of the target OS
-# ${2} - the architecture of the target OS
+# See `pkg_src_dir()`
+# ${3} - the application name
```

Comment:
Doesn't the `APP_NAME` variable contain the application name? My understanding is that `${3}` is not the application name.

Reply:
Done.